docs/reference/anonymous_usage_statistics.md

```diff
@@ -2,7 +2,7 @@
 title: Usage Statistics
 ---
 
-To help us improve the tool, by default we track event data when certain Data Context-enabled commands are run. Our [blog post from April 2020](https://greatexpectations.io/blog/anonymous-usage-statistics/) explains a little bit more about what we want to capture with usage statistics and why! The usage statistics include things like the OS and python version, and which GX features are used. You can see the exact schemas for all of our messages [here](https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/core/usage_statistics/schemas.py).
+To help us improve the tool, by default we track event data when certain Data Context-enabled commands are run. Our [blog post from April 2020]([https://greatexpectations.io/blog/anonymized-usage-statistics/](https://greatexpectations.io/blog/anonymized-usage-statistics)) explains a little bit more about what we want to capture with usage statistics and why! The usage statistics include things like the OS and python version, and which GX features are used. You can see the exact schemas for all of our messages [here](https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/core/usage_statistics/schemas.py).
```

Comment:
It looks like when you pasted the replacement link, it pasted an entire [linked text](link address) block in as the updated (link address), causing Docusaurus to consider the link broken and the netlify preview to fail.

If you strip out the excess [linked text] the update should be fine and the netlify preview will render.

```suggestion
To help us improve the tool, by default we track event data when certain Data Context-enabled commands are run. Our [blog post from April 2020](https://greatexpectations.io/blog/anonymized-usage-statistics) explains a little bit more about what we want to capture with usage statistics and why! The usage statistics include things like the OS and python version, and which GX features are used. You can see the exact schemas for all of our messages [here](https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/core/usage_statistics/schemas.py).
```